README.md

```diff
@@ -51,9 +51,10 @@ Eprint = {arXiv:1809.02167},
 * [YARP](http://www.yarp.it/): to handle the comunication with the robot;
 * [iDynTree](http://wiki.icub.org/codyco/dox/html/idyntree/html/): to handle the robot kinematics;
 * [iCubContrib](https://github.com/robotology/icub-contrib-common): to configure the modules;
+* [icub-main](https://github.com/robotology/icub-main): to handle signale processing;
```

Comment:
As far as I see `icub-main` is used only for the min-jerk-trajectory and the integrator. I didn't know hot to describe it. @S-Dafarra @GiulioRomualdi feel free to suggest better description of it.

Reply:
(I also noticed probably this include https://github.com/robotology/walking-controllers/blob/dcc2698e3d16adb5c78c973a27a359f0986ae535/src/SimplifiedModelControllers/include/WalkingControllers/SimplifiedModelControllers/ZMPController.h#L19 is not required

Reply:
Feel free to remove it and add that modification to this PR